storage/container_storage_interface/persistent-storage-csi-ovirt.adoc

```diff
@@ -0,0 +1,27 @@
+[id="persistent-storage-csi-ovirt"]
+= Red Hat Virtualization (oVirt) CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-ovirt
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Red Hat Virtualization (oVirt).
+
+Familiarity with link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVs], link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[persistent volume claims (PVCs)], link:http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html[dynamic
```

Comment:
Is recommended for what? Before using the operator? Before continuing with these docs?

Also, are all of these links to the Kubernetes docs necessary? Are there not OpenShift links that we could use instead?

Reply:
It was borrowed the wording and links from some existing storage docs, which in turn was using k8s docs as the source.

I have updated to use OCP links and to clarify that familiarity with the concepts in the links is recommended before using a CSI Operator and driver.